heroku.md

```diff
@@ -18,6 +18,12 @@ heroku create sushi
 ```bash
 git push heroku master
 ```
+## Docker
```

Comment:
Is it correct to use `##` here?

Reply:
Might be better as `###` to make it look like the rest of the sections :)